core/limits/snapshot.go

```diff
@@ -41,6 +41,7 @@ func (e *Engine) serialiseLimits() ([]byte, error) {
 				CanProposeAsset:          e.canProposeAsset,
 				GenesisLoaded:            e.genesisLoaded,
 				ProposeMarketEnabled:     e.proposeMarketEnabled,
+				ProposeSpotMarketEnabled: e.proposeSpotMarketEnabled,
```

Comment:
I think the new field is missing from `restoreLimits()`. Could do with a unittest for this.

Reply:
yep and added.